src/main/java/com/ita/if103java/ims/dto/RefillListDto.java

```diff
@@ -0,0 +1,42 @@
+package com.ita.if103java.ims.dto;
+
+import javax.validation.constraints.NotNull;
+
+public class RefillListDto {
+    @NotNull
+    private String name;
+    @NotNull
```

Comment:
there is no sense in these validations here, because this Dto is not input form from the UI. It is use only to send data from server to client